rust_src/src/keymap.rs

```diff
@@ -16,6 +17,40 @@ pub fn Ctl(c: char) -> i32 {
     (c as i32) & 0x1f
 }
 
+/// Hash table used to cache a reverse-map to speed up calls to where-is.
+declare_GC_protected_static!(where_is_cache, Qnil);
+
+/// Allows the C code to get the value of `where_is_cache`
+#[no_mangle]
+pub extern "C" fn get_where_is_cache() -> Lisp_Object {
+    unsafe { where_is_cache }
+}
+
+/// Allows the C code to set the value of `where_is_cache`
+#[no_mangle]
+pub extern "C" fn set_where_is_cache(val: Lisp_Object) -> () {
```

Comment:
There is no need to indicate an empty return here. We do that for `lisp_fn` since ALL `lisp_fn` return something.

Reply:
The `-> ()` notation is optional even for functions marked with `lisp_fn`; they're equivalent.

Reply:
It is a style we have, not a Rust requirement.

Reply:
Noted. Thanks!